### ISSUE TITLE
How would one use this library?

### ISSUE THREAD

**Issue by dwaindibbly** [miklos-molnar/M2ECS#4]
As in, compile it so it can be used in projects and stuff, like into an `a` file or a `so` file, if i knew more about clang and Makefiles i'd probably write the makefile myself.

**Comment by miklos-molnar** [comment]
Hey @dwaindibbly,

I will add CMake support in the very near future.
I have never got around to it unfortunately!

I will let you know once it is done. Should be ready in a week or so.
Created a new issue to track it: #5

In the meantime feel free to let me know any other concerns.

Thanks for your interest in my project!

**Comment by miklos-molnar** [comment]
Hey @dwaindibbly,

I have now converted the project to use CMake.

You should now be able to clone and build it easily!
